src/ImageSharp/Formats/Heif/HeifItemLink.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Six Labors.
+// Licensed under the Six Labors Split License.
+
+namespace SixLabors.ImageSharp.Formats.Heif;
+
+/// <summary>
+/// Link between <see cref="HeifItem"/> instances within the same HEIF file.
+/// </summary>
+public class HeifItemLink(Heif4CharCode type, uint sourceId)
```

Comment:
I don't think these and others need to be public.

Reply:
Indeed, made 3 classes internal